checker/func_params_count_checker.go

```diff
@@ -0,0 +1,82 @@
+package checker
```

Comment:
I think the '_checker' part of the filename is obsolete.

Reply:
Removed it.